thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/rootcause/impl/MetricEntity.java

```diff
@@ -0,0 +1,30 @@
+package com.linkedin.thirdeye.rootcause.impl;
+
+import com.linkedin.thirdeye.datalayer.dto.DatasetConfigDTO;
+import com.linkedin.thirdeye.datalayer.dto.MetricConfigDTO;
+import com.linkedin.thirdeye.rootcause.Entity;
+
+
+public class MetricEntity extends Entity {
+  public static MetricEntity fromDTO(double score, MetricConfigDTO metric, DatasetConfigDTO dataset) {
```

Comment:
multiple options again. Ultimately, we want weights provided by the user in the search context, and weights generated by pipelines and the aggregation stage in the execution context.